mule-extensions-api/src/main/java/org/mule/runtime/extension/api/runtime/process/Route.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.extension.api.runtime.process;
```

Comment:
Create package ` org.mule.runtime.extension.api.runtime.route` and put this class there, alongside `Chain`